src/dev-app/mdc-slider/mdc-slider-demo.html

```diff
@@ -1,53 +1,20 @@
-<h1>Default Slider</h1>
-Label <mat-slider #slidey aria-label="Basic slider"></mat-slider>
-{{slidey.value}}
+<p>Continuous slider</p>
```

Comment:
Usually we leave the mdc demo pretty much the same as the non-mdc demo. That way we can verify that all the same usages still work. Is there a reason you changed it so much here?

Reply:
Yeah, I wanted to test each of the different ways the slider could be used. I'll revert this once I'm done testing. For now I'll add a todo to remember to do so